README.md

```diff
@@ -12,17 +12,11 @@ Contributing is easy. To get started clone the repo.
 
 It is helpful to maintain two separate terminals for each, the configuration app and the game.
 
-Then install the configuration app's dependencies.
-
-```
-yarn install
-```
-
-You will also need to install the game depenencies
-
+Then install the app's dependencies.
+make sure you're in the /flick-it directory 
+and not in flick-it/player
 ```
-cd player
-yarn install
+run-s cinstall pinstall
```

Comment:
Could we instead put this as a script in package.json?

"setup": "run-s ..."

Then this line would just read:
yarn setup

Reply:
yes